svelte/src/infra/http/AxiosAdapter.spec.ts

```diff
@@ -0,0 +1,23 @@
+import AxiosAdapter from './AxiosAdapter';
+
+let axiosAdapter: AxiosAdapter;
+const baseUrl = 'https://jsonplaceholder.typicode.com/posts';
```

Comment:
I think it would be nice to mock the API call.
Let's say server is slow and we got a timeout, our spec would fail due to that.